skrifa/src/collections.rs

```diff
@@ -224,7 +224,7 @@ where
     }
 }
 
-#[derive(Clone)]
+#[derive(Clone, Hash)]
```

Comment:
For consistency, we’ll want to manually impl `Hash` here based on the slice data the same way we do for `PartialEq`, otherwise we may end up with situations where equal values produce different hashes due to the intermediate structure.

Reply:
You are talking about the `Hash` derive for `SmallVec`, not `Storage`, right? As far as I can tell, `Storage` doesn't implement `PartialEq`. Let me know if I misunderstood, I've updated it now.

Reply:
Oops, yes, I meant the derive on `SmallVec`. Your change looks good now.

Looks like the now unused `Hash` derive is still on `Storage`. Let's remove that and then I'll merge this.